Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -1399,6 +1393,34 @@ def filesize_formatting(s):
                   f" | Change        | {change:.1f} %  |\n")
 
 
+@check(
+    id = 'com.google.fonts/check/large_file',
+    rationale = """
+        Serving extremely large font files on Google Fonts causes usability issues. This check ensures that file sizes are reasonable.
+    """,
+    severity = 10,
+    proposal = 'https://github.com/googlefonts/fontbakery/issues/3320'
+)
+def com_google_fonts_check_large_file(font):
+    """Ensure files are not too large."""
+    WARN_SIZE = 1 * 1024 * 1024
+    FAIL_SIZE = 9 * 1024 * 1024
```

Comment:
This seems to be a check that other vendors might be interest in having on their respective profiles, but with different thresholds. Do you have a suggestion on how we could declare such overrides with custom **WARN_SIZE** and **FAIL_SIZE** values? (note: this is a merely hypothetical need)

Reply:
The easiest way I can think of is to have the profile set some default entries in the configuration object, and then we pick up those entries here. That might be an interesting facility for other checks too, and will allow end-users checking deliberately big fonts to customise the values on a per-project basis. I'll work on that idea.

Reply:
thanks! ;-)